pkg/controllers/offlineupgrade_reconcile.go

```diff
@@ -99,7 +100,12 @@ func (o *OfflineUpgradeReconciler) Reconcile(ctx context.Context, req *ctrl.Requ
 		o.Manager.finishUpgrade,
 	}
 	for _, fn := range funcs {
-		if res, err := fn(ctx); res.Requeue || err != nil {
+		if res, err := fn(ctx); verrors.IsReconcileAborted(res, err) {
+			// If Reconcile was aborted with a requeue, set the RequeueAfter interval to prevent exponential backoff
+			if err == nil {
+				res.Requeue = false
+				res.RequeueAfter = time.Duration(o.Vdb.GetUpgradeRequeueTime())
```

Comment:
I think we need to multiply the time.Duration against time.Second to ensure that the requeue time is in seconds.  It also helps clarify what unit of time we are using.

Reply:
Ok, will add this